Template.php

```diff
@@ -1177,7 +1177,7 @@ public function get_identifiers_from_url($url_sent = NULL) {
           }
           if ($this->wikiname() === 'cite web') $this->change_name_to('cite journal');  // Better template choice.  Often journal/paper
           return $this->add_if_new('osti', $match[1]);
-      } elseif (preg_match("~^https?://(?:www\.|)worldcat\.org/oclc/([0-9]+)~i", $url, $match)) {
+      } elseif (preg_match("~^https?://(?:www\.|)worldcat\.org(?:/title/\S+|)/oclc/([0-9]+)~i", $url, $match)) {
```

Comment:
Would it be clearer to use a `?` to denote the optional match?
```suggestion
      } elseif (preg_match("~^https?://(?:www\.|)worldcat\.org(?:/title/\S+)?/oclc/([0-9]+)~i", $url, $match)) {
```